http/template/fs.go

```diff
@@ -0,0 +1,50 @@
+package template
+
+import (
+	"fmt"
+	"io/fs"
+)
+
+// A mergeFS maps filepaths to fs.Open functions,
```

Comment:
What are you using for line length of comments, these look like minis?

Reply:
Just using the ["semantic linefeed"](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) encouraged by [Go docs](https://tip.golang.org/doc/comment)

Reply:
This is great, so many times i've changed a line and then had to change the next 6 lines of comments

Reply:
I think I overdo it at times,
chopping sentences up
too much.
But then,
I realize it is poetry.